.sample.env

```diff
@@ -0,0 +1 @@
+HOST=yourapp.heroku.com
```

Comment:
We need to change this to something real.

Reply:
I don't follow. A real domain name?

Reply:
Yes, either the Heroku app’s domain or we’ll want to register a domain of our own.

Reply:
I've reserved https://table-production.herokuapp.com for now. Any thoughts on a domain name? Table is just a name for the project, no idea what we should really call this!